lib/fixup.rb

```diff
@@ -124,3 +125,19 @@
     puts "#{pkg[:pkg_name].capitalize} not removed.".lightblue
   end
 end
+
+# Restart crew update if the git commit of ../lib/const.rb loaded in 
+# const.rb is different from the git commit of the potentially updated
+# ../lib/const.rb loaded here after a git pull.
+
+# Handle case of const.rb not yet defining CREW_CONST_GIT_COMMIT.
+CREW_CONST_GIT_COMMIT = '0000' unless defined?(CREW_CONST_GIT_COMMIT)
```

Comment:
Is this necessary?

Reply:
Yes. Because it isn't defined otherwise.

Reply:
If you haven't crew updated to have the variable in const.rb..

Reply:
So if the constant isn't defined yet doesn't that mean it will always restart `crew update`?  Why not just use regular variables here instead of constants to ensure the correct value is calculated?

Reply:
The variable is only unset when `const.rb` is too old to have set the variable, so really only the first time the user loads this code.

If we don't do the define check for this situation we get a NameError exception...

Reply:
So essentially, when const.rb hasn't defined the variable with the correct git commit information, we should definitely be trying to run `crew update`.